docs/tools/firmware.sh

```diff
@@ -102,30 +78,13 @@ case "$os" in
 				fi
 			fi
 		done
-		sudo modprobe -r brcmfmac || true
-		sudo modprobe brcmfmac || true
-		sudo modprobe -r hci_bcm4377 || true
-		sudo modprobe hci_bcm4377 || true
-		echo "Cleaning up"
-		sudo rm -r /tmp/apple-wifi-fw
-		echo "Keeping a copy of the firmware and the script in the EFI partition shall allow you to set up Wi-Fi again in the future by running this script or the commands told in the macOS step in Linux only, without the macOS step. Do you want to keep a copy? (y/N)"
```

Comment:
Why have the cleanup commands been removed?

Reply:
We dont use `/tmp/apple-wifi-fw` anymore, we mount the ESP to `$mountpoint`.

The firmware tar is pretty tiny (~28Mb), most users can just leave it in the ESP. If something happens to their firmware they can just run `firmware.sh` again instead of rebooting to macos to get the firmware.

Its better to just reboot the system instead of reloading modules since its more stable (at least for me).

Reply:
There should be option to remove the firmware from ESP. You shouldn't enforce this on the user. Also, reloading the modules works for me everytime. Probably give a message that try restarting if wifi still not working.

Reply:
> Also, reloading the modules works for me everytime

Actually, the modules are reloaded, its just moved to later in the script.